src/ig/IGExporter.ts

```diff
@@ -321,49 +326,76 @@ export class IGExporter {
 
     if (this.config.indexPageContent) {
       ensureDirSync(pageContentExportPath);
-      const warning = warningBlock('<!-- index.md {% comment %}', '{% endcomment %} -->', [
-        `To change the contents of this file, edit the "indexPageContent" attribute in the tank ${this.configName} file`,
-        `or provide your own index file in the ig-data${path.sep}input${path.sep}pagecontent or ig-data${path.sep}input${path.sep}pages folder.`,
-        'See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root.input'
-      ]);
-      const outputPath = path.join(pageContentExportPath, 'index.md');
-      outputFileSync(outputPath, `${warning}${this.config.indexPageContent}`);
-      this.updateOutputLog(outputPath, [this.configPath], 'generated');
-      logger.info(`Generated index.md based on "indexPageContent" in ${this.configName}.`);
 
       if (filePath) {
+        const filePathString = `${path.basename(this.igDataPath)}${path.sep}${path.relative(
+          this.igDataPath,
+          filePath
+        )}`;
+
+        let preferredFileMessage =
+          `Since a ${filePathString} file was found, the "indexPageContent" property in the ${this.configName} ` +
+          `will be ignored and an index.md file will not be generated. Remove the ${filePathString} ` +
+          `file to use the "indexPageContent" property in ${this.configName} to generate an index.md file instead.`;
```

Comment:
We really do not want to encourage the use of `indexPageContent`.  So I think this message should actually just tell the user to remove the `indexPageContent` property from the config.